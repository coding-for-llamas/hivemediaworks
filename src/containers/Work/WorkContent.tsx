const WorkContent = (): JSX.Element => (
  <>
    <div className="work-header">
      <h1 className="lg-heading">
        My
        {' '}
        <span className="text-secondary">Work</span>
      </h1>
      <h2 className="sm-heading">
        Some of the projects I've worked on.
      </h2>
    </div>
    <div className="work-projects">
      <div className="work-projects--item">
        <div className="site">
          <a className="nav_link" href="/daycare"><img src="https://www.dl.dropboxusercontent.com/s/c9mxoixh5sqnict/project1.png?dl=0" 
          alt="Screenshot of a daycare website." /></a>
          <a href="/daycare" className="btn btn-light" target="_blank" rel="noreferrer">
            Daycare
          </a>
          <a href="https://github.com/coding-for-llamas/caring-child-daycare" className="btn btn-dark" target="_blank" rel="noreferrer">
          <i className="" />
            &nbsp;
            Github
          </a>
        </div>
      </div>
    </div>
  </>
);

export default WorkContent;