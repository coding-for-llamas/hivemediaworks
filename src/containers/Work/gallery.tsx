const gallery = [
  {
    name: 'Brazilian Oil Painting',
    url: 'https://dl.dropboxusercontent.com/s/l88oozacs39lx6w/Brazilian%20Oil%20Painting%20.Folk%20Art%20Painting%20Ink%20on%20Canvas%20Dimensions_%2040_22%20X%2030_22.jpg?dl=0',
    alt: 'Painting depicting a brazilian town block with people in the streets',
    description: 'Folk Art Painting - Ink on Canvas',
  },
  {
    name: 'City Log Ink Painting',
    url: 'https://dl.dropboxusercontent.com/s/h3jn9uvcjgso0hv/City%20log%20Folk%20Art%20Painting%20Ink%20on%20Canvas%20Dimensions_%2040_22%20X%2030_22.jpg?dl=0',
    alt: 'Painting depicting a cityscape with skyscrapers',
    description: 'Folk Art Painting - Ink on Canvas',
  },
  {
    name: 'Deport',
    url: 'https://dl.dropboxusercontent.com/s/erodvq79otvgo9d/Deport.Folk%20Art%20Painting%20Ink%20on%20Canvas%20Dimensions_%2040_22%20X%2030_22.jpg?dl=0',
    alt: 'Painting depicting a port with a several ships and shipping containers stacked up',
    description: 'Folk Art Painting - Ink on Canvas',
  },
  {
    name: 'Figure in Red Cocktail Dress',
    url: 'https://dl.dropboxusercontent.com/s/zswuo5wsdif3nvm/Figure%20in%20Red%20Cocktail%20Dress%20Oil%20on%20Canvas%20Signed%20Kaladek%20Measures_%2031_22%20ht%20X%2022_22Wdth.jpg?dl=0',
    alt: 'Painting depicting a woman in a red coctail dress holding a black shawl',
    description: 'Signed Kadalek Measures - Oil on Canvas',
  },
];

export { gallery };