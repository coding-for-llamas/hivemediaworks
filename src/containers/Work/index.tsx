import WorkContent from './WorkContent';

export const Work = (): JSX.Element => {
  document.title = 'Work | Coding For Llamas';  
  return (
      <div className="Site-content">
        <main className="work-main">
          <WorkContent />
        </main>
      </div>
  );
};

export default Work;
