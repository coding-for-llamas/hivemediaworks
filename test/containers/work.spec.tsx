/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { Work } from 'src/containers/Work/index';
import WorkContent from 'src/containers/Work/WorkContent';

describe('Work', () => {
  const wrapper = shallow(<Work />);
  it('renders', () => {
    expect(wrapper.find(WorkContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(WorkContent).dive().find('div.work-header').exists()).toBe(true);
  });
  
});
