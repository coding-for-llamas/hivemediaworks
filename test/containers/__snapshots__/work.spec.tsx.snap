// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work renders 1`] = `ShallowWrapper {}`;
